bundles/remove.rst

```diff
@@ -48,8 +41,8 @@ that refers to the bundle.
 2.1 Remove Bundle Routing
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The routing for the AcmeDemoBundle can be found in ``app/config/routing_dev.yml``.
-Remove the ``_acme_demo`` entry at the bottom of this file.
+Remove routing references for the bundle, these can likely be found in either
```

Comment:
Maybe we need some minor rewording here to state that some bundles add routing config that must be removed but others don't?